s3/pkg/service/object.go

```diff
@@ -375,7 +392,7 @@ func (s *s3Service) GetObject(ctx context.Context, req *pb.GetObjectInput, strea
 
 		if bucket.ServerSideEncryption.SseType == "SSE" {
```

Comment:
Didn't get this. Do we need this part of code, as it will be executed only if SSE is not set

Reply:
correct, removed the unused code